bzt/modules/blazemeter.py

```diff
@@ -1481,6 +1484,43 @@ def post_process(self):
                 for error in session.get("errors", ()):
                     raise TaurusException(to_json(error))
 
+            # if we have captured HARs, let's download them
+            self.log.info("Services: %s", self.engine.services)
+            for service in self.engine.config.get(Service.SERV):
+                # not good to reproduce what is done inside engine
+                # but no good way to get knowledge of the service in config
+                if not isinstance(service, dict):
+                    service = {"module": service}
+                module = self.engine.instantiate_module(service.get('module', ValueError()))
+                if isinstance(module, ServiceStubCaptureHAR):
+                    self._download_logs()
+                    break
+
+    def _download_logs(self):
+        for session in self.router.master.sessions():
+            assert isinstance(session, Session)
+            for log in session.get_logs():
+                self.log.info("Downloading %s from the cloud", log['filename'])
+                cloud_dir = os.path.join(self.engine.artifacts_dir, 'cloud-artifacts')
+                if not os.path.exists(cloud_dir):
+                    os.makedirs(cloud_dir)
+                dest = os.path.join(cloud_dir, log['filename'])
+                dwn = ExceptionalDownloader()
+                with ProgressBarContext() as pbar:
+                    try:
+                        dwn.get(log['dataUrl'], dest, reporthook=pbar.download_callback)
+                    except BaseException:
+                        logging.warning("Failed to download from %s: %s", log['dataUrl'], traceback.format_exc())
```

Comment:
1. BaseException is too wide one, could TaurusNetworkError be enough?
2. It looks like duplicating of ExceptionalDownloader message
3. In my opinion we don't need traceback in warning about unsuccessful download.